deploy/install.sh

```diff
@@ -116,21 +121,27 @@ oc delete -f $KABANERO_SUBSCRIPTIONS_YAML  --ignore-not-found --selector kabaner
 
 ### CatalogSource
 
+# Install Kabanero CatalogSource
+oc apply -f $KABANERO_SUBSCRIPTIONS_YAML --selector kabanero.io/install=00-catalogsource
+
+# Check the kabanero-catalog CatalogSource pod is Running
+unset STATUS
+until [ "$STATUS" == "Running" ]
+do
+  echo "Waiting for CatalogSource kabanero-catalog pod to be ready."
+  STATUS=$(oc -n openshift-marketplace get pod -l olm.catalogSource=kabanero-catalog --output=jsonpath={.items[0].status.phase})
```

Comment:
If we're replacing a previous kabanero catalog, how can we be sure we're not getting the status of the pod that was serving the previous catalog, vs the one we just applied?